write-fonts/src/offsets.rs

```diff
@@ -159,6 +161,33 @@ where
     }
 }
 
+#[cfg(feature = "parsing")]
+impl<'a, const N: usize, T: Scalar> From<Result<&'a [BigEndian<T>], ReadError>>
+    for OffsetMarker<Vec<T>, N>
+where
+    BigEndian<T>: Copy,
+{
+    fn from(from: Result<&'a [BigEndian<T>], ReadError>) -> Self {
+        OffsetMarker::new_maybe_null(from.ok().map(|vec| vec.iter().map(|x| x.get()).collect()))
+    }
```

Comment:
This is harder to read than NullableOffsetMarker::new below. Possible to restructure to be easier to read?